runtime/src/runtime/fvm.rs

```diff
@@ -534,6 +534,11 @@ where
         Ok(batch.iter().map(|_| true).collect())
     }
 
+    #[cfg(feature = "fake-proofs")]
+    fn batch_verify_ni_seals(&self, batch: &[NISealVerifyInfo]) -> anyhow::Result<Vec<bool>> {
+        Ok(batch.iter().map(|_| true).collect())
+    }
+
```

Comment:
I think this can go, right? we're just using `batch_verify_seals` now with the NI proof types